api/v1alpha4/types.go

```diff
@@ -130,7 +130,9 @@ type PortOpts struct {
 	ProjectID           string        `json:"projectId,omitempty"`
 	SecurityGroups      *[]string     `json:"securityGroups,omitempty"`
 	AllowedAddressPairs []AddressPair `json:"allowedAddressPairs,omitempty"`
-
+ 	// DisablePortLevelSecurity disables the port security for a port on an OpenStackMachine
+	// It disables SecurityGroups on the port and removes existing AllowedAddressPairs set on the port
+	DisablePortLevelSecurity bool `json:"disablePortLevelSecurity,omitempty"`
```

Comment:
I think it makes sense to try and match existing naming schemes (OpenStack, gophercloud, OpenShift). In OpenStack, this is `disablePortSecurity`, in gophercloud, this is `PortSecurityEnabled`, in OpenShift this is `portSecurity`. Can we pick a name that already exists? I expect `disablePortSecurity` bool makes most sense.

Reply:
I agree. 
We can re-use the one at the network level (`openStackCluster.Spec.DisablePortSecurity`)

Reply:
changed to `DisablePortSecurity`